jsons/Beliefs.json

```diff
@@ -80,6 +80,7 @@
         "type": "Enhancer",
         "uniques": [
             "Only available <after adopting [Reformation]>",
+            // TODO: Implement this.
```

Comment:
Evangelism is implemented in Units.json
```
"Removes other religions when spreading religion <after adopting [Evangelism]>"
```

Maybe we should say in the comments to look there.

Reply:
``` suggestion
// Implemented as a Missionary unique. @see Units.json
```